numba/cuda/tests/cudadrv/test_linker.py

```diff
@@ -191,8 +191,8 @@ def kernel(x):
 
     @skip_with_cuda_python
     def test_linking_cu_ctypes_unsupported(self):
-        msg = ('Linking CUDA source files is not supported with the ctypes '
-               'binding')
+        msg = ('Support for fp16 or linking CUDA source files '
```

Comment:
The expected message doesn't seem to match the message added in `dispatcher.py`. Shouldn't this test be failing?

Reply:
@gmarkall the test was not failing it appears because it was not being run due to the skip_with_cuda_python decorator.